src/data/set6/champions.ts

```diff
@@ -53,6 +53,22 @@ export default {
 		},
 	},
 
+	[ChampionKey.Kassadin]: {
+		cast: (elapsedMS, spell, champion) => {
+			const target = champion.target
+			const manaReave = champion.getSpellVariable(SpellKey.ManaReave) ?? 0
+			const castTime = champion.getCurrentSpell()?.castTime ?? 0
+			if (!target) { return console.log('No target', champion.name, champion.team) }
```

Comment:
We can remove this target check completely now too! `queueProjectile` will verify and warn us if needed: https://github.com/tacticians-academy/teamfight-simulator/blob/d9fb754f7cf9e48d0bba1672f5705139e6263f04/src/game/ChampionUnit.ts#L869